### ISSUE TITLE
signalfx_org_token secret attribute does not work

### ISSUE THREAD

**Issue by Yonas-Dresen** [splunk-terraform/terraform-provider-signalfx#310]
Hello SignalFx team,

According to the documentation of the SignalFx terraform provider. In the signalfx_org_token resource you can request the secret attribute after creation.
https://registry.terraform.io/providers/splunk-terraform/signalfx/latest/docs/resources/org_token#secret

This attribute is returning 'null' at all times. When using TF_LOG=DEBUG you can clearly see the secret that the api returned, but the SignalFx provider does not store or fails to store the secret in the Terraform state file.

**Comment by mjhuber** [comment]
seeing this issue as well in version `6.14.0`.  Example:

```
resource "signalfx_org_token" "token" {
  name        = "stage-test"
  description = "stage-test"
}
```

and then looking at the state, `secret` is null but the token was created in signalfx.

```
    {
      "mode": "managed",
      "type": "signalfx_org_token",
      "name": "token",
      "provider": "provider[\"registry.terraform.io/splunk-terraform/signalfx\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "auth_scopes": [
              "API",
              "INGEST"
            ],
            "description": "stage-test",
            "disabled": false,
            "dpm_limits": [],
            "host_or_usage_limits": [],
            "id": "stage-test",
            "name": "stage-test",
            "notifications": null,
            "secret": null
          },
          "sensitive_attributes": [],
          "private": "<redacted>",
          "dependencies": [
            "data.vault_generic_secret.signalfx_creds"
          ]
        }
      ]
    },
```

**Comment by mjhuber** [comment]
update for anyone watching, this is now fixed in provider version `6.17.0`
